packages/terra-collapse/tests/jest/Collapse.test.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import IconCaretRight from 'terra-icon/lib/icon/IconCaretRight';
+import Collapse from '../../src/Collapse';
+
```

Comment:
The icon prop has a defaultProp which provides a default icon if non are specified by the user. https://github.com/cerner/terra-core/blob/199038e9d93dc96e64c428906beb0fcb63d97336/packages/terra-collapse/src/Collapse.jsx#L53